apps/app/src/client/util/apiv1-client.ts

```diff
@@ -46,7 +46,7 @@ export async function apiPost<T>(path: string, params: unknown = {}): Promise<T>
 }
 
 export async function apiPostForm<T>(path: string, formData: FormData): Promise<T> {
-  return apiPost<T>(path, formData);
+  return apiRequest<T>('postForm', path, formData);
```

Comment:
ref) https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/?tab=readme-ov-file#multipart-form-data-is-no-longer-automatically-set